website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -117,57 +117,11 @@ Don't use alerts to communicate error messages that are not caused by the user.
 </Hds::Alert>
 !!!
 
-### Ordering Alerts by color
-
-Order multiple Alerts by their importance and impact on the user, starting from `critical`, to communicate an Alert that must be resolved immediately, and descending in order of perceived impact to `neutral`.
+### Informational Alerts
```

Comment:
I see you have this section organized after "Critical alerts." Based on the existing documentation, which type of alert would be used more regularly? Would you change how the content in the Colors section is displayed?